site/2016/11/22/doctrine-mongodb-release-1.4.0.rst

```diff
@@ -0,0 +1,89 @@
+Doctrine MongoDB 1.4.0
+======================
+
+We are happy to announce the immediate availability of Doctrine MongoDB
+Abstraction Layer `1.4.0 <https://github.com/doctrine/mongodb/releases/tag/1.4.0>`__.
+
+Passing context options to the driver
+-------------------------------------
+
+With this release it's possible to driver options to the connection class, which
+will then be passed on to the MongoDB driver. For example, to pass a stream
+context with SSL context options, you could use the following code snippet:
+
+.. code-block:: php
+
+    $context = stream_context_create([
+        'ssl' => [
+            'allow_self_signed' => false,
+        ]
+    ]);
+    $connection = new \Doctrine\MongoDB\Connection(null, [], null, null, ['context' => $context]);
+
+Passing multiple expressions to logical operators
+-------------------------------------------------
+
+The ``addAnd``, ``addNor`` and ``addOr`` methods in the query and aggregation
+builders now take multiple expression objects. Instead of having to call the
+method repeatedly, you need to only call it once now:
+
+.. code-block:: php
+
+    // Before
+    $builder
+        ->addAnd($someExpression)
+        ->addAnd($otherExpression);
+
+    // After
+    $builder->addAnd($someExpression, $otherExpression);
+
+Deprecations
+------------
+
+The ``update`` and ``multiple`` methods in the query have been deprecated in
+favor of ``updateOne`` and ``updateMany``. These deprecations help people using
+ODM prepare for the next version of ODM which will utilize the new MongoDB
+library API.
+
+Bug fixes in this release
+-------------------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb/blob/master/CHANGELOG-1.4.md#140-2016-11-22>`__.
+A full list of issues and pull requests included in this release may be found in the
+`1.4.x milestone <https://github.com/doctrine/mongodb/issues?q=milestone%3A1.4.x>`__.
```

Comment:
Is there a specific 1.4.0 milestone? I would expect 1.4.x to be for things that didn't make it to 1.4.0.